javascript/javascript1/week2/weatherwear.js

```diff
@@ -0,0 +1,12 @@
+function temperatureToday(a) {
+  const temp = a;
```

Comment:
you don't have to declare another variable, you can directly use `a`.
On the other hand I would recommend giving the parameter name `temperature` not `temp` 
to avoid confusion with `temporary` 
```js
function temperatureToday(temperature) {
  if (temperature> 0 && temperature< 10) {
  ...
```

Reply:
Done!